template/en/default/setup/strings.txt.pl

```diff
@@ -387,6 +387,15 @@ END
 WARNING: Dependency graph images are not accessible.
 Delete ##dir##/.htaccess and re-run checksetup.pl.
 END
+    memcached_servers => <<'END',
```

Comment:
these should be named localconfig_memcached_servers and localconfig_memcached_namespace and be moved up near the other options -- possibly in alphabetical order